book/src/installers.md

```diff
@@ -169,14 +170,32 @@ The package will also include an npm-shrinkwrap.json file for the npm packages t
 Limitations/Caveats:
 
 * Requires a well-defined [artifact download URL][artifact-download-url]
-* [Cannot detect situations where musl-based builds are appropriate][musl] (static or dynamic) 
+* [Cannot detect situations where musl-based builds are appropriate][musl] (static or dynamic)
 * [Relies on nodejs's builtin gzip support to unpack the files, which only works with .tar.gz][unpacking]
 * Cannot run any kind of custom install logic
 
 As a result of the `.tar.gz` limitation, `cargo dist init` will prompt you to change [windows-archive][] and [unix-archive][] to ".tar.gz" if you enable the npm installer, erroring if you decline.
 
 
 
+### Homebrew
+
+> since 0.2.0
+
+This provides a [Homebrew](https://brew.sh) formula which allows users to `brew install` your package. Since it installs to the same standard paths as any other Homebrew package, it provides a simple and convenient installation method for users who already have Homebrew available. When published to a [tap](https://docs.brew.sh/Taps) (package repository), this gives your users an easy way to both install your package and to keep it up to date using `brew update` and `brew upgrade`. It fetches the same prebuilt macOS binaries as the shell installer.
```

Comment:
"Since it installs to the same standard paths as any other Homebrew package, it provides a simple and convenient installation method for users who already have Homebrew available."

as a non homebrew expert i was not sure what this meant

Reply:
I'll reword that to something like "Since it installs to a location on the user's `PATH`".